cli/command_restore.go

```diff
@@ -105,6 +114,20 @@ func addRestoreFlags(cmd *kingpin.CmdClause) {
 	cmd.Flag("ignore-permission-errors", "Ignore permission errors").BoolVar(&restoreIgnorePermissionErrors)
 	cmd.Flag("ignore-errors", "Ignore all errors").BoolVar(&restoreIgnoreErrors)
 	cmd.Flag("skip-existing", "Skip files and symlinks that exist in the output").BoolVar(&restoreIncremental)
+	cmd.Flag("shallow-restore-at-depth", "Shallow restore the directory hierarchy starting at this level (default is to deep restore the entire hierarchy.)").Short('d').IntVar(&restoreShallowAtDepth)
```

Comment:
nit: perhaps short flag is not needed for such an advanced feature.

Reply:
also, perhaps just `--shallow=N` ?

Reply:
Done.

Reply:
In 20ff248

Reply:
Done